src/action.ts

```diff
@@ -43,6 +48,7 @@ export function getConfig(): ActionConfig {
     annotations: core.getBooleanInput("annotations", { required: false }),
     verbose: core.getBooleanInput("verbose", { required: false }),
     ignoreResults: core.getBooleanInput("ignore_results", { required: false }),
+    workingDirectory: core.getInput("working_directory", { required: false }) || ".",
```

Comment:
nit: I'm not sure if it matters but I think the `|| "."` is unnecessary here now that we are checking `if (cwd)` in `src/tasks/knip.ts`.

Reply:
nice! looks good to me, let me know if there's any other way I can help finish this